NitroxModel/Logger/Log.cs

```diff
@@ -98,7 +98,11 @@ public static void Debug(object o)
         // Helping method for formatting string correctly with arguments
         private static string Format(string fmt, params object[] arg)
         {
-            return string.Format(fmt, arg);
+            if (arg.Length > 0)
```

Comment:
Does `string.Format` really break when there are no items?!

https://docs.microsoft.com/en-us/dotnet/api/system.string.format?view=netframework-3.5#System_String_Format_System_String_System_Object___

> Object[] args
An object array that contains zero or more objects to format.